test/cbmc/proofs/getFileContextFromJob/Makefile

```diff
@@ -0,0 +1,38 @@
+
+HARNESS_ENTRY = getFileContextFromJob_harness
+HARNESS_FILE = $(HARNESS_ENTRY)
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = getFileContextFromJob
+
+DEFINES += -Dstatic=""
+DEFINES += -DOTA_MAX_FILE_SIZE=2097152
```

Comment:
What is the origin of this limit?

Reply:
The maximum size of the file that can be downloaded is 16MB. The `OTA_MAX_FILE_SIZE` value signifies the maximum size of the file which can be downloaded without breaking the function under test.